### ISSUE TITLE
One of these essays has a huge number of sentences copied *exactly* from other papers: Demystifying Post-hoc Explainability for ML models

### ISSUE THREAD

**Issue by th-anderson** [Mathpix/spectra-review-paper-competition#47]
Demystifying Post-hoc Explainability for ML models

These are just some example:

Section 3 First sentence of Rule-based Explanations is copied from: https://www.aaai.org/ocs/index.php/AAAI/AAAI18/paper/view/16982

Section 3 - Last Sentence of Counterfactual Explanations is copy from https://arxiv.org/abs/1909.09369

Section 3 - Last Sentence of Representation-Based Explanations copied from https://papers.nips.cc/paper/2020/file/c74956ffb38ba48ed6ce977af6727275-Paper.pdf

Section 3 -  Summaries of Counterfactuals is copied from https://arxiv.org/abs/2009.07165

Section 4 - Last Sentence of Feature Importance Based Explanations is copied from https://paperswithcode.com/method/nam

Section 4 – Last sentence of Shapley Value Importance is copied fromhttps://arxiv.org/pdf/2002.09815.pdf

Section 4 - Patch-based Classification First Sentence is copied from https://arxiv.org/pdf/2102.05917.pdf

Section 6 - Adversarial attacks: sentence copied from https://arxiv.org/pdf/1911.02508.pdf

Section 6 - First sentence of Unjustified Counterfactual Explanations is copied from:https://arxiv.org/pdf/1907.09294.pdf

Section 6 - Last sentence of Faithfulness is copied from: https://www.nature.com/articles/s42256-019-0048-x?proof=t

Section 6 - Last sentence of Robustness is copied from https://www.groundai.com/project/on-the-robustness-of-interpretability-methods/1

**Comment by Nickil21** [comment]
Thank you for bringing this to my notice. I agree, since this was a review (summary) paper, for certain parts, I wanted to use the author(s) words/sentences verbatim, without diluting information, while of course crediting them in the review. I think I have credited all the authors and have *not claimed* that it is my finding. Maybe I should have added a quotation mark to *explicitly* convey that this is the case. If the sponsors think that this is not the right way, I am *okay* with withdrawing my submission. 

@nicodjimenez - what do you think?

**Comment by nicodjimenez** [comment]
I agree that it's not a good idea to copy sentences exactly from other papers.  This leads to stylistic inconsistencies that degrade the quality of the writing.  If you quote directly from other papers, yes you need quotations, and external links to the original sources.  

Either add quotations and direct attributions for what's copied directly, or we'll remove the submission.  Up to you @Nickil21!

**Comment by Nickil21** [comment]
@nicodjimenez: Thanks for your clarification. I think it's better that you remove my submission. I will submit a fresh one for the next contest :)
